include.js

```diff
@@ -43,6 +43,13 @@ let cards = [
     author: 'Ahmed',
     githubLink: 'https://github.com/<ahmedalhamad7>'
   },
+  {
+    artName: 'Square Bounce', // change this to the name of your artwork
+    pageLink: './Art/Ash/triangle.html', // change this
+    imageLink: './Art/Ash/triangle.gif', // change this
+    author: 'Ash V', // use your name
+    githubLink: 'https://github.com/tvermaashutosh' // change this
+  },
```

Comment:
Please remove the comments, there were only added in the README to explain what to do.

Also, please fill out the details with the name/folder path of _your contribution_.

Your folder name should be the Github username.  The file name should be the name of _your contribution_.

So, i.e. `pageLink: './Art/tvermaashutosh/index.html`.

Reply:
The entry still does not meet the necessary requirements.

a) the folder names for your contribution should match your github name
b) the links should conform to the file names, i.e. `pageLink: './Art/tvermaashutosh/index.html`